src/cnxml/jsx.ts

```diff
@@ -430,9 +435,21 @@ export declare namespace CMLNLE {
 
 export declare namespace CXLXT {
     namespace Attributes {
+        interface Born {
+            cxlxtBorn?: number
+        }
+
+        interface Died {
+            cxlxtDied?: number
+        }
+
         interface Index {
             cxlxtIndex?: string
         }
+
+        interface Name {
+            cxlxtName?: string
+        }
```

Comment:
Since those attributes are meant to be used together, this would be better modelled by a single interface (say `Name`) containing all three.